virttest/utils_test/libvirt.py

```diff
@@ -442,6 +442,12 @@ def setup_or_cleanup_nfs(is_setup, mount_dir="nfs-mount", is_mount=False,
                               "\"setsebool virt_use_nfs on\" to get "
                               "nfs access right.")
         _nfs.setup()
+        nfs_mount_info = process.run('nfsstat -m', shell=True).stdout.decode().strip().split(",")
```

Comment:
'stdout.decode()'  should be replaced by 'stdout_text' for better compatibility.